src/ethereum_spec_tools/evm_tools/utils.py

```diff
@@ -80,6 +85,8 @@ def get_module_name(forks: Any, options: Any, stdin: Any) -> Tuple[str, int]:
     """
     Get the module name and the fork block for the given state fork.
     """
+    if options.state_fork.upper() in UNSUPPORTED_FORKS:
```

Comment:
Should use [`str.casefold`](https://docs.python.org/3/library/stdtypes.html#str.casefold) for case-insensitive comparisons.

Reply:
Will update.